### PR TITLE
Documentation on readthedocs

### PR DESCRIPTION
**Pull request by saadrahim**
- Updating version number 2.6.0
- Adding readthedocs build with sphinx/breathe/exhale
- Renames the doc folder to docs to standardize across rocm projects

### CODE REVIEW

**Comment by saadrahim** [comment]
Should be merge squashed

**Comment by neon60** [comment]
Please change the target to develop.

File: CMakeLists.txt [hunk 1]
```diff
@@ -30,7 +30,7 @@ else()
 endif()
 
 # Thrust project
-project(thrust VERSION 2.5.0.0 LANGUAGES CXX)
```
Comment (neon60) [inline CMakeLists.txt]: We have this change in this pull request. Please approve that first:
https://github.com/ROCmSoftwarePlatform/rocThrust/pull/39

**Comment by neon60** [comment]
Could you solve the conflict?

**Pull request merged**